Source/WebCore/rendering/GridTrackSizingAlgorithm.cpp

```diff
@@ -270,6 +271,30 @@ LayoutUnit GridTrackSizingAlgorithm::initialGrowthLimit(const GridTrackSize& tra
     return infinity;
 }
 
+void GridTrackSizingAlgorithm::sizeTrackToFitNonSpanningItem(const GridSpan& span, LayoutUnit& singleTrackMinContentForChild, LayoutUnit& singleTrackMaxContentForChild, LayoutUnit& singleTrackMinSizeForChild, GridTrack& track)
```

Comment:
I'm not convinced this is the best name for this function. While the logic inside does seem similar to the version it overloads at a glance, I think this is doing something much different. It's taking some `LayoutUnit` arguments which represent some sort of sizes and trying to fit the track to them.